CMakeLists.txt

```diff
@@ -466,6 +466,30 @@ if (BUILD_QC_TESTS)
   add_subdirectory (test/quickcheck)
 endif(BUILD_QC_TESTS)
 
+#
+# Manual pages
+#
+find_program (ASCIIDOCTOR_EXE asciidoctor)
+if (NOT ASCIIDOCTOR_EXE)
+  message (FATAL_ERROR "Could not find asciidoctor. Try yum/gem install asciidoctor")
+endif (NOT ASCIIDOCTOR_EXE)
+add_custom_target (
+  man
+  ALL
+  COMMAND ${CMAKE_CURRENT_BINARY_DIR}/cvmfs/cvmfs_server generate-man ${CMAKE_CURRENT_BINARY_DIR}
```

Comment:
This is the problematic line not passing in Travis. Currently I am not sure how to fix that.

Second thing, on OS X it would also need to first install asciidoctor.

Reply:
Now I found the cause of Travis failing on Ubuntu (I was trying it on Fedora 27 and it worked fine).

It is because of the shebang `#!/bin/sh` in cvmfs_server that indicates usage of a strictly bourne-compatible shell; in the case of Ubuntu, dash is used. And there is no support for `declare` command.

I am using it for declaring associative arrays with `declare -A` that is supported in Bash since v4. Even on RHEL6 it's OK - ["Red Hat Enterprise Linux 6 includes version 4.1 of Bash as its default shell."](https://access.redhat.com/documentation/en-us/red_hat_enterprise_linux/6/html/migration_planning_guide/sect-migration_guide-package_changes-bash)
 It is a really nice key-value store ideal for setting options.
```
_CVMFS_SERVER_MKFS_OPTIONS=(
  [G]="set timespan for auto tags"
  [R]="require masterkeycard key"
  [V]="VOMS authorization"
   ...
)
```
The fix would be change the shebang to `#!/bin/bash` if there is no problem with it.